.vscode/settings.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
Could you remove this file? You may consider adding `.vscode` to your `.gitignore_global` file.

Reply:
@kentcdodds Please check it now!

Reply:
Please remove this file as well:

```
git rm -rf .vscode
```